jenkins-build-driver/src/main/java/org/jboss/pnc/jenkinsbuilddriver/JenkinsBuildDriver.java

```diff
@@ -42,11 +44,22 @@ public String getDriverId() {
     public boolean canBuild(BuildType buildType) {
         return BuildType.JAVA.equals(buildType);
     }
+    
+    public void init(RunningEnvironment runningEnvironment) {
+        try {
+            this.isCrumbUsed = jenkinsServerFactory.isJenkinsServerSecuredWithCSRF(runningEnvironment.getJenkinsUrl());    
+        } catch ( BuildDriverException bde) {
```

Comment:
If it safe to swallow this exception, can we continue when init fails ? Mybe we shold set the default value to isCrumbUsed ?